cclib/bridge/cclib2horton.py

```diff
@@ -80,7 +80,53 @@ def makehorton(data):
             attributes["npa_charges"] = data.atomcharges["natural"]
 
     elif hortonver == 3:
-        pass  # Populate with bridge for horton 3 in later PR
+        # In horton 3 IOData, inputs are type-checked -- thus the bridge also verifies the types.
+        # if coordinates are known; numpy.ndarray (size natom-by-3)
+        if hasattr(data, "atomcoords"):
+            attributes["atcoords"] = numpy.asanyarray(data.atomcoords)[-1]
+        # if atomic numbers are known; numpy.ndarray (size natom)
+        if hasattr(data, "atomnos"):
+            attributes["atnums"] = numpy.asanyarray(data.atomnos)
+        # if orbital coefficients known; iodata.orbitals.MolecularOrbitals
+        if hasattr(data, "mocoeffs"):
+            moattr = {
+                "kind": "restricted",
+                "norba": len(data.mocoeffs[0]),
+                "norbb": None,
+                "occs": numpy.concatenate(
+                    numpy.ones(data.homos[0]), numpy.zeros(len(data.mocoeffs[0]) - data.homos[0])
+                ),
+                "coeffs": data.mocoeffs[0],
+                "energies": None,
+                "irreps": None,
+            }
+            # and if unrestricted:
+            if len(mocoeffs) == 2:
+                moattr.kind = "unrestricted"
```

Comment:
Shouldn't these be key access rather than attribute access?

Reply:
As far as I know, these indeed are attribute access since an instance of horton3's `MolecularOrbital` class needs to be created beforehand. [This ](https://github.com/theochem/iodata/blob/b36513d162f99b57264005583701c6987037839c/iodata/orbitals.py#L30)is the code block that is relevant. I will recheck this soon to make sure.

Reply:
But in the previous expression `moattr` is created as a dictionary.

Reply:
My bad -- sorry for the confusion. I'll modify this. Thanks for catching this!